src/components/common/Actions.vue

```diff
@@ -174,6 +174,7 @@ export default class Actions extends Vue {
       // clear flag
       this.setHaveChanges(false)
     } catch (error) {
+      console.log('Error on onClickSave(): ', error)
```

Comment:
Is this the only catch block in here where the error isn't logged?

Reply:
Good thinking, I've added console.log statements to the catch blocks.  Will push a new commit shortly.

Reply:
Thanks.